### ISSUE TITLE
Vocabulário da tradução.

### ISSUE THREAD

**Issue by samcarecho** [gutomaia/inventwithpython#19]
Pessoal,

O original em inglês usa um vocabulário bem simples e acessível e isso é muito bom!
Andei lendo algumas traduções e percebi a utilização de vocabulário mais elaborado.
Também notei alterações do tempo verbal em alguns trechos traduzidos

Não seria melhor nos mantermos fieis ao estilo do texto original?

Abraço,
Sam

**Comment by stummjr** [comment]
Concordo com o @digiwise. Quando der um tempinho, vou revisar minhas traduções para ver se tem algo que deve ser melhorado. Btw, sintam-se à vontade para corrigir deslizes. :-)
